content/_help/verify-your-identity/verify-your-identity-in-person._en.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: help
+title: Verify your identity in person
+category: verify-your-identity
+order: 10
+---
+
+
+If you are unable to successfully upload your ID on Login.gov, you may be able to verify in person with our partners at the United States Postal Service (USPS). At this time, we’re only offering this option at [select USPS locations in the Washington D.C. area](https://docs.google.com/document/d/1B0f3rTeFghm1RDu4oy7RUItlTJZABg38IxjWLGaQ35g/edit#heading=h.82q7b95pjgex) and for certain partner agencies.
```

Comment:
Currently the "...select USPS locations in the Washington D.C. area" link goes to our internal Google Doc. Could you update the URL so it anchor links to the heading "Available locations in the Washington D.C. area" instead?

Reply:
This should be fixed now in ff743f5.